pages/common/age.md

```diff
@@ -0,0 +1,28 @@
+# age
+
+> A simple, modern and secure file encryption tool.
+> More information: <https://age-encryption.org>.
+
+- Generate an encrypted file that can be decrypted with a passphrase:
+
+`age --passphrase --output {{path/to/encrypted_file}} {{path/to/unencrypted_file}}`
+
+- Generate a keypair, saving the private key and the public key to an unencrypted keyfile and printing the public key to stdout:
+
+`age-keygen --output {{path/to/keyfile}}`
+
+- Encrypt a file to one or several public keys that are entered as literals:
+
+`age --output {{path/to/encrypted_file}} --recipient {{publickey1}} --recipient {{publickey2}} {{path/to/unencrypted_file}}`
```

Comment:
Here and in the `--recipients-file` example below, it would be nice for consistency to move `--output` to be the last argument.